### PR TITLE
(NOBIDS) node_jobs: allow only one bls-change-node-job per validator

### PR DESCRIPTION
**Pull request by guybrush**
```sql
CREATE TABLE IF NOT EXISTS
    node_jobs_bls_changes_validators (
        validatorindex INT NOT NULL,
        node_job_id VARCHAR(40) NOT NULL,
        PRIMARY KEY (validatorindex)
        -- we allow only one job per validator
    );
```

### CODE REVIEW

**Comment by peterbitfly** [comment]
Why a new table and not just a new column in the `validators` table?

File: db/node_jobs.go [hunk 1]
```diff
@@ -154,21 +163,47 @@ func CreateBLSToExecutionChangesNodeJob(nj *types.NodeJob) (*types.NodeJob, erro
...
-	_, err = WriterDb.Exec(`insert into node_jobs (id, type, status, data, created_time) values ($1, $2, $3, $4, now())`, nj.ID, nj.Type, nj.Status, nj.RawData)
+	tx, err := WriterDb.Beginx()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("error starting db transactions: %w", err)
+	}
+	defer tx.Rollback()
+
+	for _, idx := range indicesArr {
+		res, err := tx.Exec(`insert into node_jobs_bls_changes_validators (validatorindex, node_job_id) values($1, $2) on conflict do nothing`, idx, nj.ID)
```
Comment (peterbitfly) [inline db/node_jobs.go:184]: This will cause several thousand queries to be executed sequentially. Better to use proper batching here.

**Review by peterbitfly — CHANGES_REQUESTED** [review]

File: db/node_jobs.go (hunk 1 above)
Comment (guybrush) [reply db/node_jobs.go]: fixed in new e937dab

**Comment by guybrush** [comment]
rebased

**Pull request merged**